server/sonar-web/src/main/js/components/shared/complexity-distribution.js

```diff
@@ -57,8 +57,13 @@ export const ComplexityDistribution = React.createClass({
   },
 
   render () {
-    return <div className="overview-bar-chart" style={{ height: HEIGHT }}>
-      {this.renderBarChart()}
-    </div>;
+    // TODO remove inline styling
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)